src/cephadm/cephadm

```diff
@@ -6013,6 +6013,11 @@ def _parse_args(av):
 
 if __name__ == "__main__":
 
+    # root?
+    if os.geteuid() != 0:
+        sys.stderr.write('ERROR: cephadm should be run as root\n')
```

Comment:
what if the the user is included in the sudoers list?

Reply:
commit title was somewhat misleading. this is actually checking for the effective user id ...